pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/function/AggregationFunction.java

```diff
@@ -37,6 +37,8 @@
     PERCENTILEEST_QUANTILEDIGEST
   }
 
+  void accept(AggregationFunctionVisitorBase visitor);
```

Comment:
The AggregationFunctionVisitorBase is a no-op base class that allows creation of specialized classes overriding only relevant methods. Interface will require every derived class to implement all the methods.